### ISSUE TITLE
API Addition

### ISSUE THREAD

**Issue by austinpilz** [egold555/CorpseReborn#23]
I would find it helpful if there was an API command for a Corpse where it could return the boolean if the supplied player has a line of sight of the corpse.

Example: if (corpse.doesPlayerHaveLineOfSightOf(player))

**Comment by egold555** [comment]
This is a good idea

**Comment by egold555** [comment]
Until I figure out a way to implement this method, you can use corpse.canSee(player)
That function takes into account the render distance etc. It's used internally to see when to spawn the corpse for the player etc. It might work for you until I figure out how to make

**Comment by egold555** [comment]
In the next update I am adding isInViewDistance(player, corpse). Might help until I get around to writing this complex view distance function

**Comment by austinpilz** [comment]
That sounds pretty great! I tried to implement some method of being able to actually see the location, but I found you'll have to use RayTrace or something

**Comment by egold555** [comment]
Yes. I  will leave this marked as an enhancement, if I get around to raytracing, I will implement it :)

**Comment by austinpilz** [comment]
Thank you! Just saw you added the distance API call. Thank you :)
